src/gsCore/gsMultiPatch.h

```diff
@@ -416,6 +430,8 @@ class gsMultiPatch : public gsBoxTopology, public gsFunctionSet<T>
     const InterfaceRep & interfaceRep() const { return m_ifaces; }
     const BoundaryRep & boundaryRep() const { return m_bdr; }
     const BoundaryRep & sides() const { return m_sides; }
+
+    gsMultiPatch<T> BezierExtraction() const;
```

Comment:
rename as:
gsMultiPatch<T> extractBezier() const;

add one line of doxygen doc